thunder/executors/torch_compile.py

```diff
@@ -72,16 +72,22 @@ def make_compiled(
     region_trace = TraceCtx(None)
     region_trace.args = sorted_unique_inputs
     region_trace.kwargs = {}
-    with tracectx(region_trace):
-        for a in sorted_unique_inputs:
-            prims.unpack_trivial(a, name=a.name)
+    for a in sorted_unique_inputs:
+        region_trace.bound_symbols.append(prims.unpack_trivial.bind(a, name=a.name, output=a))
```

Comment:
I would prefer moving the name adding.
If it were not a pain to introduce, I would probably make traces own the proxies instead of the name business.